Sources/Risk/Risk.swift

```diff
@@ -41,22 +45,41 @@ public final class Risk {
         }
     }
     
-    public func publishData (cardToken: String? = nil, completion: @escaping (Result<PublishRiskData, RiskError.Publish>) -> Void) {
-        guard let fingerprintService = fingerprintService else {
-            completion(.failure(.fingerprintServiceIsNotConfigured))
-            return
+    public func publishData(cardToken: String? = nil, completion: @escaping (Result<PublishRiskData, RiskError.Publish>) -> Void) {
+        guard let _ = fingerprintService else {
```

Comment:
This is another oversight. We can bring this unwrapping back and use the non optional version on line 63 below